config_example.sh

```diff
@@ -650,6 +650,12 @@ set -x
 # More info here: https://github.com/openshift-metal3/dev-scripts/pull/1241#issuecomment-846067822
 # export BMO_WATCH_ALL_NAMESPACES="true"
 
+# For platforms that use nmstate configuration, the bond mode is set to the value.
+# The possible values are 'balance-rr', 'active-backup', 'broadcast', 'balance-xor',
+# 'balance-tlb', and 'balance-alb'.
+# The default is no bonding, so value is set to 'none'.
+# export BOND_CONFIG='none'
+
```

Comment:
It wasn't obvious to me that in order to use BOND_CONFIG, I had to also set BOND_PRIMARY_INTERFACE. Perhaps if BOND_CONFIG is not set to "none", then it should automatically set BOND_PRIMARY_INTERFACE to some value. It is also not clear what value I needed to set for BOND_PRIMARY_INTERFACE. I guessed "eth0", and that seemed to work.

Reply:
Good point, it is a bit clumsy but I wanted to keep the existing behavior for `BOND_PRIMARY_INTERFACE`.
I'll set it when` BOND_CONFIG!='none'`

Reply:
Updated